Arpan-206.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "@Arpan-206"
+project: "Bonus Lector"
+---
+
+# Bonus Lector
+
+## Summary
+
+We are going to build our own eBook Reader (like Kindle) using the Raspberry Pi with a custom OS image, books store etc.
+
+## Plan
+
```

Comment:
Please link any guides/inspo you've had! If not, please just lmk and we can continue ahead :)

Reply:
Hey, I have answered this in the PR description already

Reply:
please include links in your proposal as well thanks

Reply:
default reply moment; ok soz